### PR TITLE
Removed all LinearLayouts and formatted some lines

### PR DESCRIPTION
**Pull request by Justus-Saringer**

### CODE REVIEW

File: code/list/ListWithDifferentViews.xml [hunk 1]
```diff
@@ -15,11 +15,11 @@
 // Write Custom List Adapter
 
 class DifferentViewAdapter :
-    ListAdapter<ExampleListItems, RecyclerView.ViewHolder>(DifferentViewDiffCallback) {
+    ListAdapter&lt;ExampleListItems, RecyclerView.ViewHolder&gt;(DifferentViewDiffCallback) {
```
Comment (betafx) [inline code/list/ListWithDifferentViews.xml:19]: &lt; looks strange, is something wrong with the formatting?

Comment (Justus-Saringer) [reply code/list/ListWithDifferentViews.xml:19]: The website has problems to recognize the <>. If you push this to the website, it wont show anything.
In short, this code will be shown on the website and will not be executed.

Comment (betafx) [reply code/list/ListWithDifferentViews.xml:19]: Ah, I see. This is in the xml not in the code. Sorry, my bad :)

Comment (Justus-Saringer) [reply code/list/ListWithDifferentViews.xml:19]: Thanks for looking over it :)

**Pull request merged**